ansible/install_glance_service_init.yaml

```diff
@@ -0,0 +1,36 @@
+---
+- hosts: localhost
+  vars_files: vars/default.yaml
+  roles:
+  - oc_local
+
+  tasks:
+  - set_fact:
+      glance_yaml_dir: "{{ working_yamls_dir }}/glance"
+
+  - debug:
+      msg: "yamls will be written to {{ glance_yaml_dir }} locally"
+
+  - name: Create yaml dir
+    file:
+      path: "{{ glance_yaml_dir }}"
+      state: directory
+
+  - name: Copy files to yaml dir
+    copy:
+      src: "{{ item }}"
+      dest: "{{ glance_yaml_dir }}/"
+    with_items:
+    - "ocp/glance/service-init-cmds.yaml"
+    - "ocp/glance/service-init-job.yaml"
+
+  - name: Start glance
+    shell: |
+      set -e
+      oc apply -n openstack -f "{{ glance_yaml_dir ~ '/' ~ item }}"
```

Comment:
nit: just apply -f "{{ glance_yaml_dir }}"

K8S will sort out the ordering for you, and it only uses 1 ansible task so it's simpler and faster.